tests/e2e/e2e_test.go

```diff
@@ -891,7 +891,7 @@ func (s *IntegrationTestSuite) TestSuperfluidVoting() {
 	)
 }
 
-func (s *IntegrationTestSuite) TestCreateConcentratedLiquidityPoolVoting() {
+func (s *IntegrationTestSuite) TestCreateConcentratedLiquidityPoolVoting_And_TWAP() {
```

Comment:
Out of curiosity, why is this change paired with CL voting tests?

Reply:
The next thing we need to work on is parallelizing e2e, so pairing some logic together that doesn't require one to be run before the other makes sense imo